lib/render-html.js

```diff
@@ -1,26 +1,32 @@
-import path from 'path';
+const tryResolve = (...args) => {
+  try {
+    return require.resolve(...args);
+  } catch (err) {
+    return false;
+  }
+};
 
 export default async ({ resume, themePath }) => {
   const cwd = process.cwd();
-  let resolvedThemePath = themePath;
-  try {
-    if (themePath[0] === '.') {
-      resolvedThemePath = path.join(process.cwd(), themePath, 'index.js');
-    } else {
-      try {
-        resolvedThemePath = require.resolve(themePath, {
-          paths: process.cwd(),
-        });
-      } catch (err) {
-        resolvedThemePath = require.resolve(`jsonresume-theme-${themePath}`, {
-          paths: process.cwd(),
-        });
-      }
-    }
-  } catch (err) {
-    throw new Error(`Theme ${themePath} could not be resolved from ${cwd}`);
+  let path;
+  if (themePath[0] === '.') {
+    path = tryResolve(path.join(cwd, themePath), { paths: [cwd] });
```

Comment:
how can this work ? path is never initialized before this instruction

```js
let path;
// #=> undefined
path.join(cwd, themePath)
// #=> Uncaught TypeError: Cannot read property 'join' of undefined
```

```bash
resume export --theme . --format html /tmp/output.html
```
(node:11456) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'join' of undefined
    at _default (/usr/local/lib/node_modules/resume-cli/build/render-html.js:24:28)